back/app/models/home_page.rb

```diff
@@ -15,20 +16,86 @@
 #  banner_avatars_enabled                   :boolean          default(TRUE), not null
 #  banner_enabled                           :boolean          default(TRUE), not null
 #  banner_layout                            :string           default("full_width_banner_layout"), not null
-#  banner_signed_in_header                  :jsonb            not null
-#  banner_signed_in_text                    :jsonb            not null
-#  banner_signed_in_type                    :string           default("no_button"), not null
-#  banner_signed_in_url                     :string
-#  banner_signed_out_header                 :jsonb            not null
-#  banner_signed_out_subheader              :jsonb            not null
+#  banner_signed_in_header_multiloc         :jsonb            not null
+#  cta_signed_in_text_multiloc              :jsonb            not null
+#  cta_signed_in_type                       :string           default("no_button"), not null
+#  cta_signed_in_url                        :string
+#  banner_signed_out_header_multiloc        :jsonb            not null
+#  banner_signed_out_subheader_multiloc     :jsonb            not null
```

Comment:
Just wondering: Why do we have no `signed_in` equivalent of `banner_signed_out_subheader_multiloc` here?
Don't we want the option of a banner subheader when user is signed in?

@alexander-cit @nTraum @benfraserdesign ?

Reply:
I think we shouldn't implement it as part of this task. We just want to move the existing fields to a new table.